### PR TITLE
[5.0] PHPDocs Fixes

### PR DESCRIPTION
**Pull request by pespantelis**

### CODE REVIEW

File: src/Illuminate/Foundation/Bus/DispatchesCommands.php [hunk 1]
```diff
@@ -34,7 +34,7 @@ protected function dispatchFromArray($command, array $array)
 	 * Marshal a command and dispatch it to its appropriate handler.
 	 *
 	 * @param  mixed  $command
-	 * @param  \ArrayAccess  $array
+	 * @param  \ArrayAccess  $source
 	 * @return mixed
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Bus/DispatchesCommands.php]: We're missing the extras parameter from this docblock too.

**Comment by GrahamCampbell** [comment]
Please fix the remaining issue and squash to one commit.

**Comment by pespantelis** [comment]
@GrahamCampbell Done.

File: src/Illuminate/Foundation/Bus/DispatchesCommands.php [hunk 2]
```diff
@@ -34,7 +34,8 @@ protected function dispatchFromArray($command, array $array)
 	 * Marshal a command and dispatch it to its appropriate handler.
 	 *
 	 * @param  mixed  $command
-	 * @param  \ArrayAccess  $array
+	 * @param  \ArrayAccess  $source
+	 * @param  array  $extras
 	 * @return mixed
 	 */
 	protected function dispatchFrom($command, ArrayAccess $source, $extras = [])
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Bus/DispatchesCommands.php]: I wonder if we should add a typehint to that third param @taylorotwell.

**Pull request merged**